internal/builders/go/pkg/build.go

```diff
@@ -155,10 +157,10 @@ func (b *GoBuild) Run(dry bool) error {
 		return err
 	}
 
-	fmt.Println("dir", dir)
-	fmt.Println("binary", binary)
-	fmt.Println("command", command)
-	fmt.Println("env", envs)
+	fmt.Fprintf(Output, "dir %s \n", dir)
```

Comment:
Alternative suggestion: Use a `*log.Logger` for writing to stdout instead of `os.Writer` directly. Loggers are maybe a bit easier to work with. But maybe `os.Writer` is better if we expect users to parse the builder's output.

@laurentsimon Was the intention here that users could parse the builder's output? or is this just informational logging?

Reply:
just informational  logging